docs/interfaces/Customer.md

```diff
@@ -13,7 +13,6 @@
 - [fullName](Customer.md#fullname)
 - [id](Customer.md#id)
 - [isGuest](Customer.md#isguest)
-- [isStripeLinkAuthenticated](Customer.md#isstripelinkauthenticated)
```

Comment:
This file is managed automatically.
We normally don't touch it in a PR.
Is this change intentional?

Reply:
I didn't know it is managed automatically thats why I removed `isStripeLinkAuthenticated`
In this situation I will not modify this file.